### DISCUSSION TITLE
Icons Lists

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [franksouza183/Evolvere-Icons#126] [Ideas]
In regards to #124, it would be faster and less problematic to
use a list of monochrome files instead of actually checking for
the css classes since that way it won't have to even read and 
parse the files and the colorizers structure wouldn't have to 
be greatly changed.

**Comment by ElectronicsArchiver** [comment]
All svgs in apps/16-48?

**Reply by franksouza183** [reply]
apps - 16-48
status - 8-24
places - 16-24
mimetypes - 16-24
emblems - 8-36
devices - 16-24
categories - 16-24
animations - 22
actions - 8-48

**Reply by franksouza183** [reply]
I think it's good for build an initial list, later I can add them manually since some 64px icons use accent colors

**Reply by ElectronicsArchiver** [reply]
My current implementation #130 uses ranges, if you have outliers
you should probably first compile a list of all of them before
I add functionality to have a list of direct paths~

**Reply by franksouza183** [reply]
Is this generating the list every time we run `Colorize.js`? If so, we can use this as a helper tool for generating the final list with direct paths, and in the end the script uses only the compiled list to colorize. There's a lot of 64px icons using kde color scheme and I will need some time searching for them, this will also allow me to add and remove some of monochrome that I don't want to be colorized.

**Reply by ElectronicsArchiver** [reply]
It's not generating a list of paths, it's testing for the folder name and then the range~
